index.html

```diff
@@ -7,52 +7,202 @@
     <link href="css/index.css" rel="stylesheet" />
     <link
       rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
     />
     <title>Fusion</title>
   </head>
   <body>
-    <button
-      id="modal-btn"
-      class="btn btn-square-solid bg-cyan-900 text-white mt-16 ml-16"
-    >
-      Open Modal
-    </button>
-
-    <div id="modal" class="modal">
-      <div class="modal-cont bg-teal-600">
-        <span
-          class="close-btn text-slate-700 text-hover-white fas fa-times"
-        ></span>
-        <h2>This is a modal</h2>
-        <p>
-          sections 1.10.32 and 1.10.33] McClintock, a Latin professor at
-          Hscovered the undoubtable source. Lorem Ipsum comes from sections
-          1.10.32 and 1.10.33] McClintock, a Latin professor at Hampdehe 1914
-          translation by H. Rackham.ock
-        </p>
-      </div>
-    </div>
-    <script>
-      var modal = document.getElementById("modal");
+    <!-- full star rating -->
+
+    <!-- <div class="full-star-rating m-16 gap-3">
+      <input
+        class="full-rating-input"
+        id="star-5"
```

Comment:
I think full star rating is a part of the code. If it is not then you could just remove the commented part and refactor the code.

Reply:
yes, thank you!